Open-Code.md

```diff
@@ -0,0 +1,36 @@
+# Open code in the NHS
+
+## Introduction
+
+There are number of organisations and publications advocating open coding in its various guises within the NHS and more generally across the public sector and academia.  This page is intended as a place to collate pertinent literature and proposed/tried approaches to implementation of open code policies in situations in the NHS from individual teams up to entire organisations.
+
+There is also a significant crossover between open code and software engineering best practice, which means that they are often (always?) promoted together as an effective means to improve reproducibility, 
+
+### Glossary (DRAFT)
+
+- **Reproducible Analytics Pipelines**: analytics processes developed in open source programming languages and adhering to software engineering best practices to allow for reproducing analyses with very little effort.
```

Comment:
RAP is used later on - should this start as
```
**Reproducible Analytics Pipelines (RAP)**
```

Reply:
That's what happens when you write something backwards in the middle of the night :-)

Reply:
Done the change